code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -430,3 +430,74 @@
 	desc = "A special bulky suit that protects against hazardous, low pressure environments. Has an additional layer of armor."
 	armor = list(melee = 45, bullet = 25, laser = 30, energy = 10, bomb = 25, bio = 100, rad = 50)
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security/hos
+
+
+
+/////////////SHIELDED//////////////////////////////////
+
+/obj/item/clothing/suit/space/hardsuit/shielded
+	name = "shielded hardsuit"
+	desc = "A hardsuit with built in energy shielding. Will rapidly recharge when not under fire."
+	icon_state = "hardsuit-hos"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security/hos
+	var/current_charges = 3
+	var/max_charges = 3 //How many charges total the shielding has
```

Comment:
I wasn't really planning on using the parent type anyway. Also blocking 5 tasers is a ton.

@Zilenan91 

Admin only